.env.example

```diff
@@ -188,6 +188,10 @@ QUEUE_CONNECTION=database
 # Supported options are 'local' and 's3'.
 #FILESYSTEM_DRIVER=local
 
+# Whether or not CDash should throw an exception if it encounters
+# a problem interacting with its filesystem.
+#FILESYSTEM_THROW_EXCEPTIONS=true
```

Comment:
Does this really need to be a configuration option?  Making this configurable means that we have to add conditional logic to every single filesystem call...

Reply:
Meaning we should be checking the return value to each call of `Storage::get()`, `Storage::put()`, etc. when this setting is disabled? If so, I agree, that seems error prone.

FWIW, here's an example error message you might see from CTest with this change:
```
Unable to write file at location: inbox\/<long file name>.xml.
```

Prior to this change, CDash would instead report:
```
Failed to save submission file.
```

IMO, there's nothing too sensitive there that would justify making this behavior configurable.